android-sdk-restclient-test/src/main/res/values/styles.xml

```diff
@@ -0,0 +1,11 @@
+<resources>
+
+    <!-- Base application theme. -->
+    <style name="AppTheme" parent="Theme.AppCompat.Light.DarkActionBar">
+        <!-- Customize your theme here. -->
```

Comment:
><!-- Customize your theme here. --> [](start = 8, length = 35)

Just out of curiosity, shouldn't people modify the colors.xml file instead of this one?